JSDemos/Demos/PivotGrid/IntegratedFieldChooser/description.md

```diff
@@ -1 +1,29 @@
\ No newline at end of file
+The field chooser allows users to manage displayed pivot grid fields. You can configure the field chooser integrated into the PivotGrid or [use it as a standalone component](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/StandaloneFieldChooser/). This example demonstrates the integrated field chooser. To open the field chooser window, click the icon in the top-left corner or right-click a row or column header and select Show Field Chooser from the appeared context menu.
+
+### Enable the Integrated Field Chooser
+
+The integrated field chooser is configured in the [fieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/) object. To enable the field chooser, set the object's [enabled](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/#enabled) property to **true**. This setting adds the Field Chooser icon to the PivotGrid and the Show Field Chooser command to the context menu.
+
+### Organize Fields
+
+The field chooser window displays five field sections:
+
+- Row Fields
+- Column Fields
+- Data Fields
+- Filter Fields
+- All Fields
+
+You can use the **fieldChooser**.[layout](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/#layout) property to arrange the sections in different ways.
+
+All Fields include fields declared in the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array and auto-generated fields as shown in this demo. If you want to hide the auto-generated fields, disable the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource).[retrieveFields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/#retrieveFields) property. You can also hide any particular field if you disable its [visible](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#visible) property.
+
+You can group All Fields for hierarchical display. Specify the same [displayFolder](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#displayFolder) for the fields that should be grouped together. In this demo, the hierarchy is built on the server. 
+
+Users can drag and drop fields between the sections. When users move a field to Row, Column, Data, or Filter Fields, it is added to the corresponding pivot grid area. To do the same programmatically, specify the field's [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property. If a field is intended to be moved to the Data Fields section only, enable the field's [isMeasure](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#isMeasure) property, and vice versa: disable the **isMeasure** property for fields that should never be in the Data Fields section. In this demo, these restrictions are specified on the server.
```

Comment:
> When users move a field to Row, Column, Data, or Filter Fields, it is added to the corresponding pivot grid area.

Would not it be more clear to say 'to the Row, Column, Data, or Filter Field sections, the PivotGrid adds this field to the corresponding area.'